### PR TITLE
rm 'Sauce Labs app/Sauce Labs dashboard' references (TCD-1849)

### PR DESCRIPTION
**Pull request by kimsauce**
### Description

See:

- https://saucedev.atlassian.net/browse/TCD-1849
- https://i.wonderus.app/glossary/6316

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit/configuration/xcuitest.md [hunk 1]
```diff
@@ -394,7 +394,7 @@ devices:
 #### `id`
 <p><small>| OPTIONAL | STRING |</small></p>
 
-Request a specific device for this test suite by its ID. You can look up device IDs in the Sauce Labs app or using our [Get Devices API request](https://docs.saucelabs.com/dev/api/rdc#get-devices).
+Request a specific device for this test suite by its ID. You can look up device IDs under the device selection page or by using our [Get Devices API request](https://docs.saucelabs.com/dev/api/rdc#get-devices).
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/xcuitest.md:397]: Why "under" here? Your previous correction was "on device selection pages"... Just curious if there is a reason for the different phrasing?

File: docs/performance/transitions.md [hunk 2]
```diff
@@ -107,7 +107,7 @@ The custom `sauce:performance` command measures the performance output against a
 
 |Argument|Description|
 |---|------|
-|`name`<br/><font size="2">Required</font>|A name of the test as it would appear in the Sauce Labs application.|
+|`name`<br/><font size="2">Required</font>|A name of the test as it would appear on Sauce Labs.|
```
Comment (sweeneyskirt-sl) [inline docs/performance/transitions.md:110]: Curious why you use "on Sauce Labs" here? Seems like you say "in" for most of the other use cases.

**Review by sweeneyskirt-sl — COMMENTED** [review]
Couple minor comments regarding inconsistent phrasing, but overall, well done! A lot of these changes required contextual consideration and I think your corrections are spot-on.

File: docs/performance/transitions.md (hunk 2 above)
Comment (kimsauce) [reply docs/performance/transitions.md:110]: Fixed -- good catch, Nancy!

File: docs/testrunner-toolkit/configuration/xcuitest.md (hunk 1 above)
Comment (kimsauce) [reply docs/testrunner-toolkit/configuration/xcuitest.md:397]: Fixed -- good catch, Nancy!

**Pull request merged**